gravitee-apim-console-webui/src/management/settings/custom-user-fields/migrated/custom-user-fields-migrated.component.scss

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'sass:map';
+@use '../../../../../node_modules/@angular/material/index' as mat;
+@use '../../../../../node_modules/@gravitee/ui-particles-angular/index' as gio;
+@use '../../../../scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-theme, typography);
+:host {
+  @include gio-layout.gio-responsive-margin-container;
+}
+
+.custom-fields {
+  .card-header-with-btn {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  mat-card-title {
+    @include mat.typography-level($typography, subtitle-1);
```

Comment:
don't you think it's strange to have to change the typography only here and not do it globally? 
I suggest leaving the default one for uniqueness wdyt? . I'd like to treat this topic as a whole

Reply:
ok, lets keep it this way